sources/web/datalab/polymer/components/file-browser/file-browser.ts

```diff
@@ -459,10 +460,13 @@ class FileBrowserElement extends Polymer.Element implements DatalabPageElement {
     // Now load the sessions and update the running status of each file
     // whose id is in the session list.
     // We do not need to load sessions when not displaying the Status column,
-    // or if the current fild id has changed since the beginning of this request.
-    if (!hideStatus && fetchFileId === this.currentFile.id) {
+    if (!hideStatus) {
```

Comment:
I think if the fileManager.list call on line 436 throws an error, you might get to here without executing the fetchFileId check on line 440, in which case you might still want that check here before calling listSessionsPath. You could just add another of those if-not-the-right-fileid-then-return statements before this line. But not a big deal, it would just save one API call and only in an edge case.

Reply:
Yea I think I was mainly concerned about setting the `_fileList` incorrectly, the rest is just nice-to-haves. I'll keep this as is for now.